workspaces/linguist/.changeset/dry-teachers-tap.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage-community/plugin-linguist-backend': minor
+---
+
+**BREAKING** Removed support for what is known as the legacy backend, please use the New Backend System.
```

Comment:
lol @ "minor breaking" changes which are not considered major :P

Reply:
It's not yet 1.0, I'm following this: https://github.com/backstage/backstage/blob/master/REVIEWING.md#reviewing-changeset-bump-levels

Reply:
shall we ship as a major? It could make sense

Reply:
I would prefer to keep following the conventions I've been following so far.